assets/js/blocks/reviews/reviews-by-product/editor.scss

```diff
@@ -1,7 +1,3 @@
-.wc-block-reviews-by-product__selection {
```

Comment:
I searched and didn't see this anywhere (I also tried to see if it might be a dynamic generated class and didn't see anything). 👍